_health-care/_js/components/form-elements/ErrorableSelect.jsx

```diff
@@ -63,7 +63,7 @@ class ErrorableSelect extends React.Component {
     return (
       <div className={this.props.errorMessage ? 'usa-input-error' : undefined}>
         <label
-            className={this.props.errorMessage ? 'usa-input-error-label' : undefined}
+            className={this.props.errorMessage !== undefined ? 'usa-input-error-label' : undefined}
             htmlFor={this.selectId}>
```

Comment:
Yeah, this isn't very clear. I added the check for `undefined` because the way we add the error CSS classes is based on the presence of `errorMessage`, which isn't always necessarily correlated. For example, for DateInput I want the error CSS classes to be applied if there is an error but I don't need an error message to be displayed, so I set `errorMessage` to be an empty string, so no actual message appears, but it won't be undefined, so the proper class can be applied. I was thinking of changing the entire structure to not be dependent on `errorMessage`, since it seems like an imperfect correlation anyway, but I didn't want to tackle that in this PR. I can add a TODO to address later.